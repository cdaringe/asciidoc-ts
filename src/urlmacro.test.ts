import { describe, expect, test } from "vitest";
import { toAST } from "./mod.js";

test("Parse URL Macro", async ({ expect }) => {
  const input = `https://example.com[foo, bar]`;
  const result = toAST(input);
  expect(result).toMatchSnapshot();
});
