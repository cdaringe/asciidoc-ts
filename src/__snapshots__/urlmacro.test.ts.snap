// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Parse URL Macro 1`] = `
{
  "blocks": [
    {
      "content": [
        {
          "attributes": [
            "foo",
            "bar",
          ],
          "scheme": "https",
          "type": "UrlMacro",
          "url": "https://example.com",
        },
      ],
      "type": "Paragraph",
    },
  ],
  "type": "Document",
}
`;
